enterprise/backend/test/metabase_enterprise/sandbox/models/group_table_access_policy_test.clj

```diff
@@ -74,7 +74,7 @@
         (testing "adding new columns = not ok"
```

Comment:
I'm guessing the fix to this is the inclusion of `{:card_id msg}` in the error. If that's what the UI is expecting in order to hook the error up to the card displayed to the user should we have a test checking that that is present?

Reply:
Fixed. This was supposed to be `:message`. @flamber , can you try again